src/Cli/Exporter.cs

```diff
@@ -10,12 +11,13 @@
 using Microsoft.Extensions.Logging;
 using static Cli.Utils;
 
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
add comment that indicates this is needed so that Mock can utilize your internal functions. Was it not enough to configure internals visible to just the test project?

Reply:
No, It was giving access error.